### ISSUE TITLE
Versions of Python Supported

### ISSUE THREAD

**Issue by JordanReiter** [AlanCristhian/statically#6]
Since a lot of people still use 3.4 in various context (and support for Python 3.4 extends through 2019), you may want to indicate in documentation & setup that 3.4 isn't fully supported. Or, at the very least, running the test suite as-is in 3.4 results in a syntax error.

**Comment by AlanCristhian** [comment]
Yes, the test suit fail but the library works with 3.4. I will fix them.
